### ISSUE TITLE
Error during build new image by docker

### ISSUE THREAD

**Issue by staked-alekseyk** [ElrondNetwork/elrond-config-mainnet#47]
### What i did
I am trying to build version 1.1.62.0

1. git clone https://github.com/ElrondNetwork/elrond-config-mainnet.git
2. git checkout --force v1.1.62.0
3. docker image build . -t elrond-node-image-last -f ./docker/Dockerfile

got and error
```Step 12/25 : RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/elrond-go/go.mod | grep arwen-wasm-vm | sed 's/.* //')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
 ---> Running in 7db245da5f6b
cp: target '/lib/libwasmer_linux_amd64.so' is not a directory
The command '/bin/sh -c cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/elrond-go/go.mod | grep arwen-wasm-vm | sed 's/.* //')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so' returned a non-zero code: 1
```

i found that there are 2 versions of arwen 
```
root@cb7ab0d6cf30:/go/elrond-go/cmd/node# cat /go/elrond-go/go.mod | grep arwen-wasm-vm | sed 's/.* //' 
v1.2.19
v1.3.16
v1.3.16
```
so 2 questions:
1. what version of arwen should be used for copy lib.....so ?
2. what version of arwen should be used to build arwen binary ?

**Comment by miiu96** [comment]
I have opened a PR with a fix https://github.com/ElrondNetwork/elrond-config-mainnet/pull/49

**Comment by bogdan-rosianu** [comment]
Fixed by #49
